SIT.Manager.Avalonia/Models/LocalizationConfig.cs

```diff
@@ -0,0 +1,55 @@
+﻿using Avalonia.Markup.Xaml.Styling;
+using System.Linq;
+using System;
+
+namespace SIT.Manager.Avalonia.Models
+{
+    public static class LocalizationConfig
```

Comment:
I'm personally not a fan of marking the localization system as static and accessing it globally. This would be better added as a service and injected into classes that need it